### PR TITLE
Added documentation for --simulate

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

File: reference/masterfiles-policy-framework/controls-cf_agent.markdown [hunk 1]
```diff
@@ -8,3 +8,52 @@ This is where `body agent control` is defined. `body agent control` is where
...
+
+}
+```
+
+The simulate option takes a parameter, `diff`, `manifest`, or `manifest-full`
+which is used to determine the summary output shown at the end of the run.
+
+* `diff` - Show only things that changed during the simulated run.
+* `manifest` - Show the desired state of the host.
+* `manifest-full` - Show the desired state of the host, including changed files
```
Comment (vpodzime) [inline reference/masterfiles-policy-framework/controls-cf_agent.markdown:41]:
```suggestion
* `manifest-full` - Show all files evaluated by the simulated run (including unchanged ones)
```

File: reference/masterfiles-policy-framework/controls-cf_agent.markdown [hunk 2]
```diff
@@ -8,3 +8,52 @@ This is where `body agent control` is defined. `body agent control` is where
...
+      Use the new command line option: `--simulate=diff` or `--simulate=manifest`.
+      Please note that only files and packages promises are simulated currently.
+
+    - Added a new --simulate=manifest-full mode
+      New simulation mode that manifests all changed files as well as
+      all other files evaluated by the agent run which were not skipped
+      (by file selection rules) (CFE-3506)
+
+#### Notes
+* Supported on Linux for `files` and `packages` type promises
```
Comment (vpodzime) [inline reference/masterfiles-policy-framework/controls-cf_agent.markdown:54]: Not sure if this is the Linux-only part is true.

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/399/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/399/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-399/output/_site/

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/400/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/400/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-400/output/_site/

**Pull request merged**